ncbi/src/main/java/org/reactome/release/Utilities.java

```diff
@@ -0,0 +1,84 @@
+package org.reactome.release;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardOpenOption;
+import java.util.*;
+
+/**
+ * General Utilities class.  This class contains methods needed for general manipulation of data or common operations
+ * used by the data export process.
+ * @author jweiser
+ */
+public class Utilities {
+	/**
+	 * Takes a set of Strings and splits it into a list of the requested number of sub-sets.  If an ordered set, order
+	 * is preserved.  If the set does not split evenly into the number of set-sets requested, the remainder elements
+	 * will be added one element each to the beginning sets of the returned list.
+	 * @param set Set of Strings to split
+	 * @param numOfSubSets Number of sub-sets to return
+	 * @return List of String sub-sets
+	 */
+	public static List<Set<String>> splitSet(Set<String> set, int numOfSubSets) {
+		int subSetSize = set.size() / numOfSubSets ;
```

Comment:
Why is the `numOfSubSets` variable needed?

Reply:
This is to let the method know into how many sub-sets the set that's being passed should be split.